WordPress/Classes/Models/Theme.m

```diff
@@ -57,6 +57,15 @@ + (Theme *)createOrUpdateThemeFromDictionary:(NSDictionary *)themeInfo
     return theme;
 }
 
+#pragma mark - CoreData helpers
+
++ (NSString *)entityName
+{
+    return NSStringFromClass([self class]);
```

Comment:
Nice touch! i wonder if we shouldn't move this `entityName` method to a `NSManagedObject` category, so that it's available for all of the Core Data entities.